yaml2ics.py

```diff
@@ -25,11 +25,34 @@
 }
 
 
-def strfexception(exdate):
-    if isinstance(exdate, datetime.datetime):
-        return datetime.datetime.strftime(exdate, "%Y%m%dT%H%M%S")
-    elif isinstance(exdate, datetime.date):
-        return datetime.datetime.strftime(exdate, "%Y%m%d")
+def datetime2utc(date):
+    if isinstance(date, datetime.datetime):
+        return datetime.datetime.strftime(date, "%Y%m%dT%H%M%S")
+    elif isinstance(date, datetime.date):
+        return datetime.datetime.strftime(date, "%Y%m%d")
+
+
+# See RFC2445, 4.8.5 REcurrence Component Properties
+# This function can be used to add a list of e.g. exception dates (EXDATE) or recurrence dates (RDATE)
+# to a reoccurring event
+def add_recurrence_property(
```

Comment:
Splitting it out exceptions and extra date into a common function makes the code more readable. Nice implementation.